test/wasm_test.cc

```diff
@@ -195,6 +196,13 @@ TEST_P(TestVm, AlwaysApplyCanary) {
             auto wasm_handle_comp =
                 createWasm(vm_key, source, plugin_comp, wasm_handle_factory_comp,
                            wasm_handle_clone_factory_for_canary, false);
+            // Validate that canarying is cached for the first variant.
```

Comment:
I think it is worth mentioning that the reason why canarying of the first variant was cached is that it happened during creating the baseline plugin above.